### DISCUSSION TITLE
LSP per language config to unify setting

### DISCUSSION THREAD

**Discussion by meijieru** [LunarVim/LunarVim#2189] [Q&A] [answered]
How to avoid generating the corresponding `after/ftplugin/xxx.lua` in `/home/jmei/.local/share/chezmoi/dot_config/lvim/after`?

**Comment by kylo252** [comment, marked as answer]
Any server that's currently overridden, won't be generated

```lua
vim.list_extend(lvim.lsp.override, { "pyright" })
```
ref: https://www.lunarvim.org/languages/#server-setup
